packages/playwright-core/src/server/chromium/crBrowser.ts

```diff
@@ -359,6 +359,20 @@ export class CRBrowserContext extends BrowserContext {
     await Promise.all(promises);
   }
 
+  private _shouldSetDownloadBehavior(): boolean {
+    if (this._browser.options.name === 'electron') {
+      // Chromium v124 correlates to Electron v30 which is the first version that has https://github.com/electron/electron/pull/41718 backported.
```

Comment:
Note: This expect users to have the latest minor/patch release - probably better but still risky?

Reply:
Shall we wait a bit for electron 31 instead? It seems like we are not in a rush, given that existing users with old electron version won't get the support anyway.